glib/src/variant_iter.rs

```diff
@@ -85,7 +85,7 @@ impl<'a> VariantStrIter<'a> {
             ffi::g_variant_get_child(
                 self.variant.to_glib_none().0,
                 i,
-                s as *const u8 as *const i8,
+                s as *const u8 as *const _,
                 &p,
                 std::ptr::null::<i8>(),
```

Comment:
@decathorpe This one is not a problem?

Reply:
Would it work to just do `s as *const _`?

Reply:
Nevermind, I see this was covered in the commit message.

Reply:
@sdroege apparently not ... which data types and which pointers can be converted to which others implicitly is a bit beyond me, but probably NULL is always NULL.

Reply:
Regarding `std::ptr::null::<i8>()` I suspect it doesn't matter because it's part of the varargs parameters, which inherently aren't type checked.

Reply:
Ah, of course. Thanks :)